packages/validation/package.json

```diff
@@ -53,6 +53,8 @@
     "isomorphic-fetch": "3.0.0",
     "lodash": "4.17.21",
     "tiny-invariant": "1.2.0",
+    "tiny-glob": "0.2.9",
```

Comment:
perhaps these should move to devDependencies since they are only used during tests?

Reply:
Yeah, let's move them to devDeps.